### ISSUE TITLE
Can you give me the models which are trained?

### ISSUE THREAD

**Issue by xiaoyuge16** [QData/deepWordBug#2]
3Q~

**Comment by xiaoyuge16** [comment]
When I run train.py, I find that the accuracy is zero.
![snipaste_2018-08-18_20-49-52](https://user-images.githubusercontent.com/23721563/44299437-5bdc8c80-a328-11e8-9bc0-5dd32b559ac3.png)

why?
I install torch is V0.41.
I try to install torch 0.2, but I don't know which whl is suitable?(https://pytorch.org/previous-versions/)
![snipaste_2018-08-18_20-43-07](https://user-images.githubusercontent.com/23721563/44299392-5b8fc180-a327-11e8-9adc-c7793480955f.png)

**Comment by xiaoyuge16** [comment]
when I intall torch0.2, it shows segment fault.
what should I do ?
I can't run this code, I feel very sad. Can you help me? For example, can you give me a dockerfile in which your code is successly running?
I run this code in a machine which can't access to Internet, So can you give me a dockerfile or other good method to run this code?
3Q very much!

**Comment by xiaoyuge16** [comment]
My machine use cuda8.0

**Comment by gaoji7777** [comment]
#3 
Good to know that you've solved the problem. Thank you for your interest in our project! If you have some new progress, feel free to contact us on Github.
